packages/flutter/lib/src/scheduler/ticker.dart

```diff
@@ -55,11 +60,15 @@ abstract class TickerProvider {
 /// are called.
 ///
 /// By convention, the [start] and [stop] methods are used by the ticker's
-/// consumer, and the [muted] property is controlled by the [TickerProvider]
-/// that created the ticker.
+/// consumer (for example, an [AnimationController]), and the [muted] property
+/// is controlled by the [TickerProvider] that created the ticker (for example,
+/// a [State] that uses [TickerProviderStateMixin] to silence the ticker when
+/// the state's subtree is disabled as defined by [TickerMode]).
 ///
 /// Tickers are driven by the [SchedulerBinding]. See
 /// [SchedulerBinding.scheduleFrameCallback].
```

Comment:
```suggestion
/// See also:
///
/// * [TickerProvider], which has instructions for obtaining a ticker.
/// * [SchedulerBinding] and its [SchedulerBinding.scheduleFrameCallback]
///   method, used to drive the ticker.
```

Reply:
Sure, good idea. Made a version that's a bit shorter:

```dart
/// See also:
/// 
/// * [TickerProvider], for obtaining a ticker.
/// * [SchedulerBinding.scheduleFrameCallback], which drives tickers.
```